src/index.js

```diff
@@ -127,6 +127,9 @@ class EventsCoreAPI {
    */
   getAllProviders (consumerOrgId, fetchEventMetadata = false, providerOptions = {}) {
     const headers = {}
+    if (providerOptions && providerOptions.providerMetadataIds && providerOptions.providerMetadataId) {
```

Comment:
Maybe we could use optional chaining, not sure what the standard is in the project however

Reply:
did not work in prev node versions, so im not using it here